docs/rules/no-unsafe-finally.md

```diff
@@ -0,0 +1,99 @@
+# Disable Unsafe Statements in `finally` Block (no-unsafe-finally)
+
+Using control flow operations in a `finally` block may lead to unexpected behavior. For example:
```

Comment:
Explain the unexpected behavior using words. What happens when you use a `return` in `finally`? What happens when you throw an error in `finally`?